SA/content/tool_research/flo.typ

```diff
@@ -0,0 +1,27 @@
+= flo
+
+flo is a visual and functional programming language. It features a programming
+environment that is based on blocks connected using cables. The corresponding
+compiler is able to convert the visual arrangements and connections into
+Haskell code.
+
+#figure(

```

Comment:
I'd try to make references to figures in the text. It probably won't have any impact on grading though

Reply:
That makes sense, I think I've read them somewhere as well. Added references in the text